modules/claims_api/lib/benefits_documents/configuration.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require 'common/client/configuration/rest'
+require 'lighthouse/auth/client_credentials/jwt_generator'
+require 'lighthouse/auth/client_credentials/service'
+
+module BenefitsDocuments
+  ##
+  # HTTP client configuration for the {DirectDeposit::Service},
+  # sets the base path, the base request headers, and a service name for breakers and metrics.
+  #
+  class Configuration < Common::Client::Configuration::REST
+    API_SCOPES = %w[documents.read documents.write].freeze
+    TOKEN_PATH = 'oauth2/benefits-documents/system/v1/token'
+
+    ##
+    # @return [Config::Options] Settings for direct_deposit API.
+    #
+    def settings
+      Settings.lighthouse.benefits_documents
```

Comment:
these would need to be specific to our teams credentials, so we'll want to create new ones. These values are actually not even in the settings file yet because this code is so new. Originally I referenced the Benefits Claims one that Jerek had created.